app/test/block_size_test.go

```diff
@@ -0,0 +1,290 @@
+package app_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/suite"
+
+	cosmosnet "github.com/cosmos/cosmos-sdk/testutil/network"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	"github.com/celestiaorg/celestia-app/testutil/network"
+	"github.com/celestiaorg/celestia-app/x/payment/types"
+	"github.com/celestiaorg/nmt/namespace"
+	abci "github.com/tendermint/tendermint/abci/types"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+	"github.com/tendermint/tendermint/pkg/consts"
+	rpctypes "github.com/tendermint/tendermint/rpc/coretypes"
+	coretypes "github.com/tendermint/tendermint/types"
+)
+
+type IntegrationTestSuite struct {
+	suite.Suite
+
+	cfg      cosmosnet.Config
+	encCfg   encoding.EncodingConfig
+	network  *cosmosnet.Network
+	kr       keyring.Keyring
+	accounts []string
+}
+
+func NewIntegrationTestSuite(cfg cosmosnet.Config) *IntegrationTestSuite {
+	return &IntegrationTestSuite{cfg: cfg}
+}
+
+func (s *IntegrationTestSuite) SetupSuite() {
+	s.T().Log("setting up integration test suite")
+
+	if testing.Short() {
```

Comment:
Is using this better than checking an environment variable?

Reply:
imo, yes, as its builtin. Open to other solutions tho